centreon/www/class/centreonDB.class.php

```diff
@@ -203,7 +203,9 @@ public function __construct($db = self::LABEL_DB_CONFIGURATION, $retry = 3)
             );
         } catch (Exception $e) {
             if (php_sapi_name() !== "cli") {
-                $this->displayConnectionErrorPage($e->getMessage());
+                $this->displayConnectionErrorPage(
+                    $e->getCode() === 2002 ? "Unable to connect to database" : $e->getMessage()
```

Comment:
Are you sure this is not a string ?
Because PDO used to return string codes.
```suggestion
                    (int) $e->getCode() === 2002 ? "Unable to connect to database" : $e->getMessage()
```